exercises/pangram/src/main/java/Pangrams.java

```diff
@@ -0,0 +1,6 @@
+public class Pangrams {
```

Comment:
I have to chuckle at this.  Somewhere in the way back I think there is a discussion between me and @sit with a different, but somehow similar flavor.  😄